Unknown sensors

mboisson opened the pull request:
This PR adds support for sensors of unknown device class. It also records the units of measurement. This can be used for example to address issue #53, or for any other type of sensor that is not explicitly supported

mboisson committed:
Create genericComponentUnknownSensor.groovy

mboisson committed:
Update HA parent.groovy

Add support for sensors with unknown device classes

ymerj commented:
Since this pertains to sensors, wouldn't it better to handle de device class definition in the sensor case instead of in the general section? Maybe other domain could use the same approach in future developement and would use a different way of defining the device class.

mboisson committed:
Update genericComponentUnknownSensor.groovy

Also add the string version of the value for sensors that do not report numbers, but status

mboisson committed:
moved device class inference to the sensor section

mboisson commented:
> Since this pertains to sensors, wouldn't it better to handle de device class definition in the sensor case instead of in the general section? Maybe other domain could use the same approach in future developement and would use a different way of defining the device class.

This is done. I also added a string type value for sensors that do not report numbers but rather status.

tomwpublic committed:
version info

tomwpublic committed:
version info

tomwpublic committed:
Update packageManifest.json

tomwpublic reviewed (approved):
I added version information and made a couple of cosmetic edits.  Please check for correctness @mboisson .  Thank you for the contribution!
